src/AppInstallerCommonCore/Manifest/ManifestSchemaValidation.cpp

```diff
@@ -99,82 +99,68 @@ namespace AppInstaller::Manifest::YamlParser
 
     Json::Value LoadSchemaDoc(const ManifestVer& manifestVersion, ManifestTypeEnum manifestType)
     {
-        std::string schemaStr;
+        int idx = 0;

```

Comment:
1. Is there any guarantee that no resource has idx 0?
2. I couldn't find anything explicitly about the type, but I saw that it is integer and only uses 16 bytes. Maybe use `WORD` instead of `int`? In any case, I had to look up what was the type of these IDX_ macros to be sure an int was appropriate. Can you add a comment for the future?

Reply:
Added comment for clarification